build-system/tasks/process-3p-github-pr.js

```diff
@@ -188,7 +188,7 @@ function getIssues(opt_page) {
 /**
  * API call to get all changed files of a pull request.
  * @param {!Object} pr
- * @return {*} TODO(#23582): Specify return type
+ * @return {?Array<string>}
```

Comment:
lets fix line 168, couldnt comment on it.

Reply:
Good catch. Done.